cmd/noms/noms_log.go

```diff
@@ -66,6 +69,13 @@ func runLog(args []string) int {
 	useColor = shouldUseColor()
 	cfg := config.NewResolver()
 
+	tz, err := locationFromTimezoneArg(tzName, nil)
```

Comment:
Doesn't our new flag system handle this for us?

Reply:
I don't believe it does, but I did need to add a flag definition for "tz" in the kingpin flag definition code in noms.go.